### DISCUSSION TITLE
I have an issues when installing foundry

### DISCUSSION THREAD

**Discussion by Minkeee** [Cyfrin/foundry-full-course-cu#1768] [Q&A] [answered]
When i input this on my wsl terminal
```
curl -L https://foundry.paradigm.xyz | bash
```
I always get this 
```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:06 --:--:--     0curl: (6) Could not resolve host: foundry.paradigm.xyz
```

**Comment by Pedrog75** [comment]
Have you tried to check is your wsl can access internet with this command: `ping google.com` ?

**Reply by Minkeee** [reply]
Sorry I haven't, can you explain how is it works?

**Reply by Minkeee** [reply]
When i try to input this command
```
ping google.com
```
it returns 
```
ping: google.com: Temporary failure in name resolution
```

**Reply by Minkeee** [reply]
I try to use this tutorial https://www.youtube.com/watch?v=DEWH9idV00M , to use change the DNS but after that, but its still not working.

**Comment by EngrPips** [comment]
Hello @Minkeee, Have you resolved this already?

**Reply by 0x4Nayan04** [reply]
@Minkeee is the issue is solved I am facing the same error ?

**Reply by Minkeee** [reply]
No, I still got an error. I try to use VPN but its still not working.

**Comment by EngrPips** [comment, marked as answer]
I think this StackOverflow post, `https://stackoverflow.com/questions/62314789/no-internet-connection-on-wsl-ubuntu-windows-subsystem-for-linux` might help resolve this issue. Let us know the outcome of trying out what the post suggests.

**Reply by 0x4Nayan04** [reply]
Yes thanks i tried it and it works

**Reply by Minkeee** [reply]
It works guys! Thank you so much!

**Reply by EngrPips** [reply]
Awesome, I am glad you guys figured it out. Keep crushing it guys.
